tests/end_to_end/definitions/kcf_custom_image_definitions.py

```diff
@@ -0,0 +1,116 @@
+import os
+from datetime import timedelta
+
+from dotenv import load_dotenv
+
+import featureform as ff
+
+
+FILE_DIRECTORY = os.getenv("FEATUREFORM_TEST_PATH", "")
+featureform_location = os.path.dirname(os.path.dirname(FILE_DIRECTORY))
+env_file_path = os.path.join(featureform_location, ".env")
+load_dotenv(env_file_path)
+
+def get_random_string():
+    import random
+    import string
+    return "".join(random.choice(string.ascii_lowercase) for _ in range(10))
+
+def save_to_file(filename, data):
+    global FILE_DIRECTORY
+    with open(f"{FILE_DIRECTORY}/{filename}", "w+") as f:
+        f.write(data)
+
+VERSION=get_random_string()
+os.environ["TEST_CASE_VERSION"]=VERSION
+
+FEATURE_NAME = f"ice_cream_feature_{VERSION}"
+FEATURE_VARIANT = "lag_features"
+TRAININGSET_NAME = f"ice_cream_training_{VERSION}"
+TRAININGSET_VARIANT = "lag_features"
+
+FEATURE_SERVING = f"farm:farm"
+VERSIONS = f"{FEATURE_NAME},{FEATURE_VARIANT}:{TRAININGSET_NAME},{TRAININGSET_VARIANT}"
+
+save_to_file("feature.txt", FEATURE_SERVING)
+save_to_file("version.txt", VERSIONS)
+
+
+# Start of Featureform Definitions
+azure_blob = ff.register_blob_store(
+    name=f"k8s_blob_store_{VERSION}",
+    account_name= os.getenv("AZURE_ACCOUNT_NAME", None),
+    account_key= os.getenv("AZURE_ACCOUNT_KEY", None),
+    container_name= os.getenv("AZURE_CONTAINER_NAME", None),
+    root_path="testing/ff",
+)
+
+redis = ff.register_redis(
+    name = f"redis-quickstart_{VERSION}",
+    host="quickstart-redis", # The internal dns name for redis
+    port=6379,
+    description = "A Redis deployment we created for the Featureform quickstart"
+)
+
+k8s = ff.register_k8s(
+    name=f"k8s_{VERSION}",
+    store=azure_blob,
+    docker_image=os.getenv("K8S_RUNNER_BASE_IMAGE", "featureformcom/k8s_runner:latest")
```

Comment:
should you default to the scikit image?

Reply:
Nah I want that to be the base image. Itll default to the latest version when you run it locally